TestAssets/TestProjects/LightupLib/project.json

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
I'm talking about LightupLibrary bringing in ITS own dependency.  Loading 1 assembly usually works fine (MSBuild does this).  But bringing it that assemblies dependency closure is usually a problem.